tests/e2e/specs/performance/payment-methods.spec.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+const { merchant } = require( '@woocommerce/e2e-utils' );
+
+/**
+ * Internal dependencies
+ */
+import { setupProductCheckout } from '../../utils/payments';
+import { shopperWCP, merchantWCP } from '../../utils';
+import {
+	recreatePerformanceFile,
+	logPerformanceResult,
+	measureCheckoutMetrics,
+	averageMetrics,
+} from '../../utils/performance';
+
+describe( 'Checkout page performance', () => {
+	beforeAll( async () => {
+		// Start a new file for every run.
+		recreatePerformanceFile();
+	} );
+
+	describe( 'Stripe element', () => {
```

Comment:
The reason I did this was because I wanted to have a clean file per performance run. At the same time, I wanted each scenario (Stripe element, UPE, WooPay without UPE, etc) to have its own setup. As a result, I did 
```
describe // first level to always wipe the file before we start a run
     describe // Stripe element setup
     describe // UPE setup
     describe // WooPay without UPE setup
```

Looking at this again, I wonder if I can mix `beforeAll` and `beforeEach` with 1 layer of `describe`, such that wiping the files execute only once. 👀